crates/core/tedge_mapper/src/c8y/converter.rs

```diff
@@ -834,11 +881,33 @@ pub fn get_child_id_from_topic(topic: &str) -> Result<Option<String>, Conversion
     }
 }
 
+pub fn get_child_id_from_event_topic(topic: &str) -> Result<Option<String>, ConversionError> {
+    match topic.strip_prefix("tedge/events/").map(String::from) {
+        Some(child_ev_topic) => {
+            let v: Vec<&str> = child_ev_topic.splitn(2, '/').collect();
+            if v.len() >= 2 {
+                let maybe_child_id = v[1].to_string();
+                if !maybe_child_id.is_empty() {
+                    return Ok(Some(maybe_child_id));
+                } else {
+                    return Err(ConversionError::InvalidChildId { id: maybe_child_id });
+                }
+            }
+            Ok(None)
+        }
+        option => Ok(option),
```

Comment:
```suggestion
        None => Ok(None),
```

Just a small nitpick here: I'd rather be explicit here. This reduces the mental load when reading the code, as one can clearly see what is returned without having to look at the top of the `match` block again.

Reply:
Yeah, fixed it.